src/context/watchLater-context.js

```diff
@@ -0,0 +1,57 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios";
+
+const WatchLaterContext = createContext();
```

Comment:
Suggestion: You could merge all three context into one, as they are all related to video - [ watch, like, videoList]

Reply:
I think keeping separate is good to simplied and keep there function in there context, and its readable